setup.py

```diff
@@ -21,9 +54,8 @@
         'Programming Language :: Python',
         'Framework :: Django',
     ],
-    install_requires=[
-        'shortuuid>=0.4.3,<1.0.0'
-    ],
+    install_requires=load_requirements('requirements/requirements.txt', 'requirements/requirements-django.txt'),
```

Comment:
I'd love to get some clarity on this too, it seems like the sort of thing we ought to standardize. I've seen all of the following used as ways to install a given repo (locally, not from pip):

- `make <install_command>`
- `setup.py install`
- `pip install -r requirements.txt`